lab/src/Microsoft.ServiceModel.Syndication/tests/TestFeeds/LoadAndWrite.xml

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<feed xml:base="http://mypage.com/" xmlns="http://www.w3.org/2005/Atom">
+  <title type="text">Microsoft News</title>
+  <subtitle type="text">&lt;div&gt;Most recent news from Microsoft&lt;/div&gt;</subtitle>
+  <id>123FeedID</id>
+  <updated>2017-06-19T11:52:39-07:00</updated>
+  <logo>http://2.bp.blogspot.com/-NA5Jb-64eUg/URx8CSdcj_I/AAAAAAAAAUo/eCx0irI0rq0/s1600/bg_Microsoft_logo3-20120824073001907469-620x349.jpg</logo>
+  <author>
+    <email>jerry@microsoft.com</email>
+  </author>
+  <link rel="alternate" href="http://www.microsoft.com/news" />
+  <CustomElement xmlns="">asd</CustomElement>
+  <entry>
+    <id>uuid:9b7dbcd0-86ed-4e86-9049-bb480d3fda65;id=1</id>
+    <title type="text">SyndicationFeed released for .net Core</title>
+    <updated>2017-06-19T18:52:39Z</updated>
+    <link rel="alternate" href="http://microsoft.com/news/path" />
+    <content type="text">A lot of text describing the release of .net core feature</content>
+  </entry>
+</feed>
```

Comment:
Maybe call this file something like "SimpleAtomFeed.xml" so it can be reused for other tests too in the future.